src/main/java/org/apache/commons/compress/archivers/tar/TarArchiveEntry.java

```diff
@@ -1634,19 +1627,19 @@ private void processPaxHeader(final String key, final String val, final Map<Stri
             setLinkName(val);
             break;
         case "gid":
-            setGroupId(Long.parseLong(val));
+            setGroupId(ParsingUtils.parseLongValue(val, 10));
```

Comment:
The PR now add this "10" magic number in many places. Is it ever NOT 10? Or, do we need a specialized version of `parseLongValue` as `parseLongValueBase10`?